plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/names/FreshNames.kt

```diff
@@ -21,11 +21,19 @@ data class AnonymousName(val n: Int) : MangledName {
         get() = "anonymous\$$n"
 }
 
-data object ReturnVariableName : MangledName {
+/**
+ * Name for variables that should *only* be used in signatures, not in
```

Comment:
I think a description piece is missing (?)

Reply:
Thanks, fixed